astropy/cosmology/core.py

```diff
@@ -1677,7 +1677,7 @@ def efunc(self, z):
         return np.sqrt(zp1 ** 2 * ((Or * zp1 + Om0) * zp1 + Ok0) + Ode0)
 
     def inv_efunc(self, z):
-        r""" Function used to calculate :math:`\frac{1}{H_z}`.
+        r""" Function used to calculate :math:`\\frac{1}{H_z}`.
```

Comment:
This is where I first noticed it not rendering correctly:
http://docs.astropy.org/en/latest/api/astropy.cosmology.FlatLambdaCDM.html

so I changed the rest of them not noticing the "raw" string.